src/SerializeResult.cc

```diff
@@ -73,6 +73,10 @@ sos::Object WrapParseResultAST(snowcrash::ParseResult<snowcrash::Blueprint>& blu
         }
     }
 
+    for (auto& warning : context.warnings) {
```

Comment:
what about:
```
std::copy(context.warnings.begin(), context.warnings.end(), std::back_inserter(blueprint.report.warnings));
```

It is supported long time in C++ and we can avoid auto until we check support in all compilers